### ISSUE TITLE
Add link to dandi jupyterhub onto main page or dandiset page

### ISSUE THREAD

**Issue by rly** [dandi/dandiarchive#1028]
In our NWB outreach, we sometimes use or refer to the dandi jupyterhub, but as far as I can see, there is no link or easy way to get to the dandihub except if you happen to know the URL. Can this link be added to the main page and/or from a dandiset page? I vaguely remember this existing previously.

**Comment by satra** [comment]
good idea. i think we were still in do not make it too explicit mode in the past. but at this point this should be completely fine.
